### PR TITLE
2019-12-03

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-12-04-bitrise-ios.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+status: publish
+categories: [Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[This post is NOT sponsored, I just like the service.]
+
+Bitrise provides Continuos Integration and Continuos Deployment for iOS and Android applications. Whenever you decide (for example after each new commit) Bitrise will build the run, run the tests and either notify you of problems or if you want it caan automatically releaase a new build to the store.
```
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: caan - > can

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: releaase - > release

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: Something wrong with yoor A key?

File: _posts/2019-12-04-bitrise-ios.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+- Workflow
+
+The workflow defines the action that will be taken by Bitrise each time a new build is requested (by pusing a specific branch, opening a pull requedst of whatever is defined in the Triggers section). The basic actions are already configured for you, but if you need a custom aciton you wil need to add it to the queue. For example in our project we are using swiftgen to generate some Swift files automatically so we added a new script to generate the files after running the Cocoapod installation step.
```
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: pusing - > pushing

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: requedst - > request

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: wil - > will

File: _posts/2019-12-04-bitrise-ios.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+- Code Signing
+
+Here you can add the certificates and provisioning profiles if you want to set up code signing manually or use an apple account to set it up automatically
```
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: Missing "." at the end

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: Add them everywhere...

File: _posts/2019-12-04-bitrise-ios.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+Opening a pull request targeting a specific branch (or any bramnch)
+Pushing a new git tag
+
+- Stack
+
+Here you can define which version of Xcode and mac OS will be used for the build
+
+- Bitrise.yml
+
+\This is the generated configuration file with everything you have set up in the previous screens. You can also edit this file directly.
```
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: What's the slash for

File: _posts/2019-12-04-bitrise-ios.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+- Add the webhook
+In the code section you will find the webhook set up. Bitrise can set it up automatically or it can be setup manually by pasting the provided bitrise webhook url inside the appropriate section on Github, Gitlab or Bitbucket.
+For Gitlab go to Settings -> Web Hooks -> add the url to the URL field and select the events you want to ping the webhook with.
+Bitbucket Settings -> Webhooks -> Create Webhook -> add the url to the URL field
+Github Settings -> Webhook -> Add webhook -> add the url inside the payload on the URL field
+
+## Conclusion
+
+While we are not using Bitrise for Continuos Deployment, all our deployments still happen manually, we can be confident that each time we push a new commit or open a pull request the app will build and passes all tests so anybody can clone it without worries.
```
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: passes - > pass?

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+I'm a huge Star Wars fan so view the whole article under this lens. If you're not a Star Wars fan there isn't really much in this game that has not been done bette before in other games, but it being Star Wars makes it special for us.
```
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: bette - > better

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+## PLUS
+
+- Great graphics
+
+The first Star Wars open world game that has good graphics for the time, but only because we didn't get a single SW open world game in the past 10 years. Thanks EA...
+
+- Great gameplay
+
+The puzzles are fun and you don't need to read a guide to understand them (most of the times). The combat is dynamic, with the right level of difficulty (haven't tried the hardest difficulties yet though).
```
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:24]: Babyyyyy

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+- Few bugs
+
+I'm used to experience lots of bugs in a newly released game. Usually during the first month they are pretty unbearable in most AAA games. Here I did not find any major issue. What I found were really minor problems that did not affect the enjoyment of the game at all.
```
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: issue - > issues

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+- Confusing maps
+
+That's not necessarely a downside, only a personal preference. A lot of people will love them. And I'm sure that will time they will start making more sense.
+
+There is also a map screen which helps, but often the maps are on multiple vertical levels adding to the confusion and making the visual map not really super useful in the beginning.
+
+- Timing puzzles
+
+For a few puzzles (the large majority do not present this problem) you need to time your clicks to be very precise or you will fail it. Even (on my estimation) a 300ms delay will make you fail so this is really bad for people either with disabilities or who wants to enjoy the game without too much stress. This happens even in story mode since it is not related to the combat part of the game but to puzzles.
```
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: it - > them

Comment (MsSlacker) [reply _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: wants - > want

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+## Conclusion
```
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:50]: Nothing about story or the characters?

File: _posts/2019-12-06-website-categories.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+image:
+image2:
+author: Valentino Urbano
+---
+
+I'm mostly writing about technology and programming in particular. I plan to continue this trend in the future.
+
+That doesn't mean that my blog is only limited to those topics as the main argument. It represents what I like so it can and will shift over time since I don't like to have different blogs for different areas. It should be the one place where anyone ([me included][1]) can find (almost) everything that I write online.
+
+I try to vary and write about a lot of different topics. I do not want my blog to be e thematic. I want it to be a reflection of who i am and what i find interesting in life.
```
Comment (MsSlacker) [inline _posts/2019-12-06-website-categories.md]: E?

Comment (MsSlacker) [reply _posts/2019-12-06-website-categories.md]: CAPITALISE I

File: _posts/2019-12-06-website-categories.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+This way though it is hard for someone who is interested in only one topic to follow only what they're interested in since I post daily and I might go a long time talking only about one topic and ignoring the rest.
+
+I have categories that should help with that, but I have them being very general (by choice). For example I have a category called programming, but I do not have a category called unity of swift even though I cover both topics with more than just a few articles.
+
+I still don't know how to solve this problem effectively since I do not want to pollute the categories page with 15+ different entries. I feel like that would ruin the whole point of that a category should be. The number of categories should be limited and adding one should only be done after thinking over it for a while.
```
Comment (MsSlacker) [inline _posts/2019-12-06-website-categories.md]: whole point of that a category - > missing what

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-12-04-bitrise-ios.md (hunk 1 above)
Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: Continuous Integration.......

File: _posts/2019-12-04-bitrise-ios.md (hunk 4 above)
Comment (valeIT) [reply _posts/2019-12-04-bitrise-ios.md]: Got automatically inserted by the editor, no idea why

File: _posts/2019-12-04-bitrise-ios.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-12-04-bitrise-ios.md]: > Something wrong with yoor A key?

Something is wrong with every key. It is called buttefly keyboard... =(

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md (hunk 7 above)
Comment (valeIT) [reply _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:24]: What a worthy comment... :P

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md (hunk 10 above)
Comment (valeIT) [reply _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:50]: Can't really say anything about the story until I finish it. Same for the characters I guess. ..?

File: _posts/2019-12-06-website-categories.md (hunk 11 above)
Comment (valeIT) [reply _posts/2019-12-06-website-categories.md]: > E?

what?

Comment (MsSlacker) [reply _posts/2019-12-06-website-categories.md]: "I do not want my blog to be e thematic." what's e thematic?

File: _posts/2019-12-06-website-categories.md [hunk 13]
```diff
@@ -15,14 +15,14 @@ I'm mostly writing about technology and programming in particular. I plan to con
 
 That doesn't mean that my blog is only limited to those topics as the main argument. It represents what I like so it can and will shift over time since I don't like to have different blogs for different areas. It should be the one place where anyone ([me included][1]) can find (almost) everything that I write online.
 
-I try to vary and write about a lot of different topics. I do not want my blog to be e thematic. I want it to be a reflection of who i am and what i find interesting in life.
+I try to vary and write about a lot of different topics. I do not want my blog to be e thematic. I want it to be a reflection of who I am and what i find interesting in life.
```
Comment (MsSlacker) [inline _posts/2019-12-06-website-categories.md]: what i find - > this I also needs capitalization!

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-12-06-website-categories.md (hunk 11 above)
Comment (valeIT) [reply _posts/2019-12-06-website-categories.md]: oh ok

**Comment by valeIT** [comment]
Wait until #104  is merged before merging this.

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md (hunk 10 above)
Comment (MsSlacker) [reply _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:50]: Well you finished it!

**Review by MsSlacker — APPROVED** [review]

**Review by MsSlacker — APPROVED** [review]

**Review by MsSlacker — DISMISSED** [review]
Starting most of them with "here" is not really good tho.

File: _posts/2019-12-04-bitrise-ios.md [hunk 14]
```diff
@@ -52,11 +52,15 @@ Here you can define which version of the OS and development tools (Xcode for iOS
 
 - Bitrise.yml
 
-This is the generated configuration file with everything you have set up in the previous screens. You can also edit this file directly if you prefer.
+This is the generated configuration file with everything you have set up in the previous screens. You can also edit this file directly if you prefer. You can also download this file and later uploading for a different application if you want to save and reuse most of the configuration. Usually the steps to build an application are similar so we tend to keep one file for each platform as a base that we can later tweak for each specific application.
```
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: uploading - > upload

File: _posts/2019-12-04-bitrise-ios.md [hunk 15]
```diff
@@ -52,11 +52,15 @@ Here you can define which version of the OS and development tools (Xcode for iOS
...
 
 - Add the webhook
 
-In the code section you will find the webhook set-up area. Bitrise can set it up automatically or it can be setup manually by pasting the provided bitrise webhook url inside the appropriate section on Github, Gitlab or Bitbucket. The automatic set up doesn't always work so if you receive an error you will be fo0rced to proceed manually. The process is pretty straightfoward: copy the Bitrise link and paste it in the relevant section on your Git host service.
+A webhook will enable the automation features of Bitrise so you don't have to start each build manually, but Bitrise gets notified of changes (push, pull requests, tags,..) by your git provider. You can se it up from the code section -> webhook set-up area.
```
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: se - > set

File: _posts/2019-12-04-bitrise-ios.md [hunk 16]
```diff
@@ -52,11 +52,15 @@ Here you can define which version of the OS and development tools (Xcode for iOS
...
 
 - Add the webhook
 
-In the code section you will find the webhook set-up area. Bitrise can set it up automatically or it can be setup manually by pasting the provided bitrise webhook url inside the appropriate section on Github, Gitlab or Bitbucket. The automatic set up doesn't always work so if you receive an error you will be fo0rced to proceed manually. The process is pretty straightfoward: copy the Bitrise link and paste it in the relevant section on your Git host service.
+A webhook will enable the automation features of Bitrise so you don't have to start each build manually, but Bitrise gets notified of changes (push, pull requests, tags,..) by your git provider. You can se it up from the code section -> webhook set-up area. 
+
+Bitrise can set it up automatically, or it can be setup manually, by pasting the provided bitrise webhook url inside the appropriate section on Github, Gitlab or Bitbucket. The automatic set up doesn't always work so if you receive an error you will be fo0rced to proceed manually.
```
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: fo0rced? Lol

**Review by MsSlacker — CHANGES_REQUESTED** [review]

**Comment by valeIT** [comment]
is it ok?

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md [hunk 17]
```diff
@@ -0,0 +1,52 @@
...
+
+The first Star Wars open world game that has good graphics for the time, but only because we didn't get a single SW open world game in the past 10 years. Thanks EA...
+
+- Great gameplay
+
+The puzzles are fun and you don't need to read a guide to understand them (most of the times). The combat is dynamic, with the right level of difficulty (haven't tried the hardest difficulties yet though).
+
+- Few bugs
+
+I'm used to experience lots of bugs in a newly released game. Usually during the first month they are pretty unbearable in most AAA games. Here I did not find any major issues. What I found were really minor problems that did not affect the enjoyment of the game at all.
```
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: experience - > experiencing

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md [hunk 18]
```diff
@@ -0,0 +1,52 @@
...
+
+- Monetization
+
+Finally an EA game that does not have predatory microtransactions or a subcription. What a breath of fresh air.
```
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: subcription - > subscription

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md [hunk 19]
```diff
@@ -0,0 +1,52 @@
...
+
+- Star Wars Story And Characters
+
+If you're a Star Wars fan you're going to like both the story and the characters. You can expect a typical Star Wars Jedi story alongside funny and relatable characters - the humor is on point as well. The best character is the little droid on the cover and will be with you for the whole gameplay.
+
+## MINUS
+
+- Confusing maps
+
+That's not necessarely a downside, only a personal preference. A lot of people will love them. And I'm sure that will time they will start making more sense.
```
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: will - > with

**Comment by valeIT** [comment]
@MsSlacker

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**